### PR TITLE
docs fix [WIP]

### PR DESCRIPTION
**Pull request by rodriguezsergio**
This is mostly just spell check and grammar fixes.

Also included:
- an attempt at style consistency throughout each page
- fixes for clarity
- broken url fixes
- small updates in documentation for text that no longer holds true (e.g. only available workflow is magnetar)

### CODE REVIEW

File: docs/src/hugo/content/documentation/cli.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ This is typically very useful when building out (or upgrading) your own unit tha
   c0e4281e9c9d  dev      howdy-http        0.38
 ```
 
-The `unit list` subcommand comes with a variety of switches and options, for your convenience:
+The `units list` subcommand comes with a variety of switches and options, for your convenience:
```
Comment (timperrett) [inline docs/src/hugo/content/documentation/cli.md]: FYI, both plural and singular versions of every subcommand work here

**Comment by timperrett** [comment]
Excellent @rodriguezsergio - thank you for your diligence and consistency!

**Comment by rodriguezsergio** [comment]
:+1: I may have another PR incoming for the operator side of things once I see a real life Nelson deployment at CompStak.

**Comment by timperrett** [comment]
@rodriguezsergio you carry on good sir; im deploying the changes from this PR now so you'll see them on the site in like ~30 mins once the build and release robots do their thing. Really excited to get feedback from folks seeing the project with fresh eyes, so keep it coming! 🙏🏻

**Pull request merged**